_search-plugins/sql/sql/functions.md

```diff
@@ -225,3 +225,63 @@ The results contain documents that match the wildcard expression:
 | account_number | address
 :--- | :---
 1 | 880 Holmes Lane
+
+## Nested query
+
+The nested function is used in the `SELECT` clause to unnest nested object type collections. The nested collection is flattened and a cartesian product is returned when querying against two nested objects.
```

Comment:
When used in the SELECT clause?

Reply:
This document doesn't mention anything about the WHERE clause.  Maybe we can add that too?